api/envoy/type/v3/proxy_protocol.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package envoy.type.v3;
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.type.v3";
+option java_outer_classname = "ProxyProtocolProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Proxy Protocol]
+
+message ProxyProtocolConfig {
```

Comment:
Is this a core type or does it belong somewhere like `envoy.config.core`? I prefer to keep the type hierarchy for fundamental types.

The reason that there is no v4 is that this does not reference any deprecated fields or other types that have transitively deprecated fields.

Reply:
Yeah, I wasn't sure where to put it. Taking another look, the types already here probably would be more widely used than the proxy proto config. I can move to `envoy.config.core`

Reply:
+1, agreed on moving it. Thank you.

/wait